src/test/integration/requirements.txt

```diff
@@ -0,0 +1,6 @@
+certifi==2018.11.29
+chardet==3.0.4
+idna==2.7
+PyYAML==3.13
```

Comment:
pyyaml should be upgraded to >=4.2b1 due to security vulnerability issue. Also, I'd recommend upgrading all of these packages to the latest version if possible.